CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## To be released
+- Updates to automated system test config and scripts.
```

Comment:
I don't think we need this line in the CHANGELOG. We really only need an entry when it's some change that would concern an "end developer".  I think we could use something like "Added automated system tests" here though.